content/en/docs/concepts/configuration/configmap.md

```diff
@@ -221,6 +221,74 @@ in a Pod:
    in the specified environment variables.
 
 This is an example of defining a ConfigMap as a pod environment variable:
+
+The following ConfigMap (myconfigmap.yaml) stores two properties: username and access_level:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: myconfigmap
+data:
+  username: "k8s-admin"
+  access_level: "1"
+```
+
+The following command will create the ConfigMap object:
+
+```shell
+kubectl apply -f myconfigmap.yaml
+```
+The output is similar to this:
+```console
+configmap/myconfigmap created
+```
+
+The following command will produce a JSON object that contains the ConfigMap's key-value pairs:
+
+```shell
+kubectl get configmap myconfigmap -o jsonpath='{.data}' | jq
+```
+The output is similar to this:
+```console
+{
+  username: "k8s-admin"
+  access_level: "1"
+}
```

Comment:
These paragraphs are irrelevant to the subject.
There is nothing wrong in it, but reading them takes time, maintaining them takes time.
This is not a tutorial page, so you don't need to detail all the verification steps.